xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -16,6 +16,8 @@
 #include "cores/VideoPlayer/VideoRenderers/RenderManager.h"
 #include "windowing/GraphicContext.h"
 #include "guilib/GUIFontManager.h"
+#include "guilib/GUIComponent.h"
+#include "guilib/StereoscopicsManager.h"
```

Comment:
this kind of globals hack is rather ugly. player is not supposed to depend on guilib.

Reply:
so how to get the 3d mode of the video file？ any advice?

Reply:
It is passed with VideoPicture from VideoPlayerVideo to RenderManager